<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFlowsheetViewer</class>
 <widget class="QDialog" name="CFlowsheetViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flowsheet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../DyssolGUI/Resources.qrc">
    <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>620</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelImage">
         <property name="text">
          <string>[Flowsheet image]</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as image or text</string>
   </property>
  </action>
  <action name="actionFitToWindow">
   <property name="text">
    <string>&amp;Fit to window</string>
   </property>
   <property name="toolTip">
    <string>Resize image to fit the window</string>
   </property>
  </action>
  <action name="actionShowPorts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;ports</string>
   </property>
   <property name="toolTip">
    <string>Whether to show ports</string>
   </property>
  </action>
  <action name="actionHorizontalLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Horizontal</string>
   </property>
   <property name="toolTip">
    <string>Use horizontal layout</string>
   </property>
  </action>
  <action name="actionVerticalLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vertical</string>
   </property>
   <property name="toolTip">
    <string>Use vertical layout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../DyssolGUI/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
